app/mirage/config.js

```diff
@@ -207,6 +207,31 @@ export function testConfig() {
         return response;
     });
 
+    this.get('/posts/:id', function (db, request) {
+        let {id} = request.params;
+        let post = db.posts.find(id);
+
+        return {
+            posts: [post]
+        };
+    });
+
+    this.put('/posts/:id', function (db, request) {
+        let [attrs] = JSON.parse(request.requestBody).posts;
+        let post;
+
+        if (isBlank(attrs.slug) && !isBlank(attrs.title)) {
+            attrs.slug = attrs.title.dasherize();
+        }
+
+        // NOTE: this does not use the post factory to fill in blank fields
+        post = db.posts.insert(attrs);
```

Comment:
No, that's just replicates something the server does when a new post is created, I think we can leave it out for updates.